aqueduct_client/aqueduct_client.py

```diff
@@ -249,7 +251,7 @@ def get_pipeline_results_dataframe(self, execution_id):
         url = response.json()['url']
 
         # get the data from the url
-        results_url_resp = requests.get(url)
+        results_url_resp = self._session.get(url)
```

Comment:
Why is this not a path like other calls to the server?

Reply:
Also note, this request will now have the apikey in the header. It might be more correct to do requests.get here instead of using the session object.